src/test/java/com/decide/DecideTest.java

```diff
@@ -582,40 +582,53 @@ public void LIC14TestNegative() {
 
     @Test
     /**
-     * Tests LIC12 for false boundary points where the number of points 
-     * can not be lower than 3, k_pts has to be positive, and both length1
-     * and length2 has to be positive.
+     * Requirements: See `LIC12` documentation
+     * Contract:
+     *      Precondition: numPoints can not be less than 3, numPoints = 2,
+     *                    or k_pts can not be lower than 0, k_pts = -1,
```

Comment:
As pointed out in #98, these preconditions should probably have the format `x is less than y` instead of stating the boundary outright.

Reply:
yes, should be fixed now!